### ISSUE TITLE
Do not see plume for methane engines

### ISSUE THREAD

**Issue by Bornholio** [jbengtson/RealISRU#21]
Using normal RO setup no real/stock plume visible, starting troubleshooting.

**Comment by Bornholio** [comment]
Loaded today with no issue on plume.  Only changes to DRE configs.
Did notice still has no Methane fuel bar.

**Comment by Bornholio** [comment]
http://imgur.com/n6aACuU
http://imgur.com/Zwtwifz

example Fix for fuel bars  /master/GameData/RealISRU/Parts/MMParts/Mars_ISRU.cfg
line 37, 75 and 134
```
		@PROPELLANT[LiquidFuel]
		{
			@name = LqdMethane
			@ratio = 0.427
			@DrawGauge = True
}
```

**Comment by stratochief66** [comment]
Yeah, I forgot to add the gauge. Please PR at, we'd gladly accept the fix.

**Comment by Bornholio** [comment]
closed by #22
